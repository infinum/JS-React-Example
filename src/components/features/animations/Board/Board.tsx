import { Box, BoxProps } from '@chakra-ui/react';
import { FC } from 'react';

export const Board: FC<BoxProps> = (props) => {
	return (
		<Box
			sx={{
				backgroundColor: '#ffffff',
				opacity: 1,
				backgroundImage:
					'linear-gradient(#c9c9c9 2px, transparent 2px), linear-gradient(90deg, #c9c9c9 2px, transparent 2px), linear-gradient(#c9c9c9 1px, transparent 1px), linear-gradient(90deg, #c9c9c9 1px, #ffffff 1px)',
				backgroundSize: '50px 50px, 50px 50px, 10px 10px, 10px 10px',
				backgroundPosition: '-2px -2px, -2px -2px, -1px -1px, -1px -1px',
			}}
			{...props}
		/>
	);
};
