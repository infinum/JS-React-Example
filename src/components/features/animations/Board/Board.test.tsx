import { axe } from 'jest-axe';
import { render } from 'test-utils';
import { Board } from './Board';

describe('Board', () => {
	it('should matches snapshot', () => {
		const { asFragment } = render(<Board />);

		expect(asFragment()).toMatchSnapshot();
	});

	it('should be accessible', async () => {
		const { container } = render(<Board />);

		const results = await axe(container);

		expect(results).toHaveNoViolations();
	});
});
