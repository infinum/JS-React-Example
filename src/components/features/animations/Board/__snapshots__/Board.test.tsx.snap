// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board should matches snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  background-color: #ffffff;
  opacity: 1;
  background-image: linear-gradient(#c9c9c9 2px, var(--chakra-colors-transparent) 2px),linear-gradient(90deg, #c9c9c9 2px, var(--chakra-colors-transparent) 2px),linear-gradient(#c9c9c9 1px, var(--chakra-colors-transparent) 1px),linear-gradient(90deg, #c9c9c9 1px, #ffffff 1px);
  -webkit-background-size: 50px 50px,50px 50px,10px 10px,10px 10px;
  background-size: 50px 50px,50px 50px,10px 10px,10px 10px;
  -webkit-background-position: -2px -2px,-2px -2px,-1px -1px,-1px -1px;
  background-position: -2px -2px,-2px -2px,-1px -1px,-1px -1px;
}

<div
    class="emotion-0"
  />
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;
